dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/checkpoint/CheckpointKafkaSaslPlainTest.java

```diff
@@ -60,6 +61,7 @@
  * Basic test using a kafka broker with TLS enabled
  */
 @RunWith(FATRunner.class)
+@MaximumJavaLevel(javaLevel = 21)
```

Comment:
Since this issue only affects Java 23, and Java 22 is still supported, we should only exclude these tests from Java 23.  Same comment for all sections.